### PR TITLE
Add nodejs.dev link to get-involved -> learning

### PR DESCRIPTION
**Pull request by watson**
For details see https://medium.com/@nodejs/node-js-selected-by-google-for-dev-top-level-domain-early-access-6b0c5f9205f8

Two questions that needs to be ok'ed before merging:

- I'm not sure if this is the expected way to "handle" translations. I looked at some of the other commits to this file and saw this approach used.
- I came up with link text and description quickly. If someone have a better text for these let me know 😄

### CODE REVIEW

File: locale/uk/get-involved/index.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ layout: contribute.hbs
 ## Навчання
 
 - [Офіційна довідкова документація по API](/api) описує Node API.
+- [Node.js Guides](https://nodejs.dev) Guides to Node.js development.
```
Comment (vsemozhetbyt) [inline locale/uk/get-involved/index.md]:
```suggestion
- [Node.js Guides](https://nodejs.dev) містить посібники з розробки на Node.js.
```

Comment (watson) [reply locale/uk/get-involved/index.md]: Thanks 😊

**Comment by ghost** [comment]
@celyes: Please trans for your Arabic.

**Comment by ghost** [comment]
@nodejs/localization：Please trans for all of your own languages if possible and free.

**Review by ghost — APPROVED** [review]
LGTM in English version

**Comment by celyes** [comment]
@Maledong 
Updated file : `locale/ar/get-involved/index.md`
See https://github.com/nodejs/nodejs.org/pull/2100

**Review by ZYSzys — COMMENTED** [review]
AFAIK, nodejs.dev was designed as our new website, not just for learning. It was in development rapidly now.

Maybe it’s not so suitable to add it here, IMHO.

**Comment by fhemberger** [comment]
I'd put that on hold as well.

File: locale/uk/get-involved/index.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ layout: contribute.hbs
 ## Навчання
 
 - [Офіційна довідкова документація по API](/api) описує Node API.
+- [Node.js Guides](https://nodejs.dev) містить посібники з розробки на Node.js.
```
Comment (haoxins) [inline locale/uk/get-involved/index.md:21]: LGTM in the Chinese version

**Comment by haoxins** [comment]
LGTM in the Chinese version

File: locale/fr/get-involved/index.md [hunk 3]
```diff
@@ -26,6 +26,8 @@ layout: contribute.hbs
 
 - [La documentation officielle de l'API de référence](/api) détaille l'API Node (en anglais).
 
+- [Node.js Guides](https://nodejs.dev) Guides to Node.js development.
```
Comment (aymen94) [inline locale/fr/get-involved/index.md:29]:
```suggestion
- [Node.js Guides](https://nodejs.dev) pour le développement de Node.js.
```

File: locale/en/get-involved/index.md [hunk 4]
```diff
@@ -18,6 +18,7 @@ layout: contribute.hbs
 ## Learning
 
 - [Official API reference documentation](/api) details the Node.js API.
+- [Node.js Guides](https://nodejs.dev) Guides to Node.js development.
```
Comment (aymen94) [inline locale/en/get-involved/index.md:21]:
```suggestion
- [Node.js Guides](https://nodejs.dev) to development.
```

Comment (aymen94) [reply locale/en/get-involved/index.md:21]: Duplication of _Guides_ and  _Node.js_

File: locale/ca/get-involved/index.md [hunk 5]
```diff
@@ -18,6 +18,7 @@ layout: contribute.hbs
 ## Aprenentatge
 
 - La [Documentació oficial de l'API](/api) detalla l'API de Node.
+- [Node.js Guides](https://nodejs.dev) Guides to Node.js development.
```
Comment (aymen94) [inline locale/ca/get-involved/index.md]: - [Node.js Guides](https://nodejs.dev) to development.

Comment (aymen94) [inline locale/ca/get-involved/index.md]:
```suggestion
- [Node.js Guides](https://nodejs.dev) to development.
```

File: locale/fa/get-involved/index.md [hunk 6]
```diff
@@ -18,6 +18,7 @@ layout: contribute.hbs
 ## Learning
 
 - [Official API reference documentation](/api) details the Node API.
+- [Node.js Guides](https://nodejs.dev) Guides to Node.js development.
```
Comment (aymen94) [inline locale/fa/get-involved/index.md]:
```suggestion
- [Node.js Guides](https://nodejs.dev) to development.
```

**Comment by Maledong** [comment]
@aymen94：I don't think it is duplicated and no need to modify, because "Node.js Guides" is the title and " Guides to Node.js development." is the detailled explaination. Since this is still on hold, so I just leave my comment here for others to have a review :)

**Review by ghost — CHANGES_REQUESTED** [review]
I don't think it is duplicated and no need to modify, because "Node.js Guides" is the title and " Guides to Node.js development." is the detailled explaination. Since this is still on hold, so I just leave my comment here for others to have a review.

@aymen94

**Review by ghost — CHANGES_REQUESTED** [review]
A lot of things to be changed and conflicts should be resolved.

**Comment by yous** [comment]
In this branch, there is no `locale/ko/get-involved/index.md` file, so I can't update the corresponding part.

**Comment by MylesBorins** [comment]
@watson would you be able to rebase?

**Comment by Trott** [comment]
@watson I'm interested in seeing this get pushed across the finish line. Are you invested enough to resolve the merge conflicts? If so, awesome! If not, I'll close this and (maybe) open my own alternative.

**Comment by MylesBorins** [comment]
quick ping about conflicts

**Review by willin — APPROVED** [review]

**Comment by MylesBorins** [comment]
This has been un touched in quite a bit and is stale. Closing. Please feel free to lmk if you would like it reopened

**Pull request closed without merging**